package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@edx/frontend-app-course-authoring",
+  "version": "0.1.0",
+  "description": "Frontend application template",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/edx/frontend-app-course-authoring.git"
+  },
+  "browserslist": [
+    "last 2 versions",
+    "ie 11"
+  ],
+  "scripts": {
+    "build": "fedx-scripts webpack",
+    "i18n_extract": "BABEL_ENV=i18n fedx-scripts babel src --quiet > /dev/null",
+    "is-es5": "es-check es5 ./dist/*.js",
+    "lint": "fedx-scripts eslint --ext .js --ext .jsx .",
+    "snapshot": "fedx-scripts jest --updateSnapshot",
+    "start": "fedx-scripts webpack-dev-server --progress",
+    "test": "fedx-scripts jest --coverage --passWithNoTests"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint"
+    }
+  },
+  "author": "edX",
+  "license": "AGPL-3.0",
+  "homepage": "https://github.com/edx/frontend-app-course-authoring#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/edx/frontend-app-course-authoring/issues"
+  },
+  "dependencies": {
+    "@edx/frontend-component-footer": "10.0.9",
+    "@edx/frontend-component-header": "2.0.5",
+    "@edx/frontend-platform": "1.1.14",
+    "@edx/paragon": "7.2.1",
```

Comment:
It doesn't have to be in this PR, but as a follow-up we can bump all these dependencies and such up to their most recent versions.  Paragon, for instance, is on v9.  I'm slowly updating the template app but haven't done some of them yet...